jest.config.js

```diff
@@ -0,0 +1,14 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+    testEnvironmentOptions: {
+        customExportConditions: ['node'],
+    },
+    moduleFileExtensions: ['js', 'vue'],
```

Comment:
if You add ts in the transformer I think it should be listed here too? or not?

Reply:
done https://github.com/demos-europe/demosplan-ui/pull/51/commits/15da49a4112a04e02695c86f47e245e31790c5ae